client/java-spring-boot-autoconfigure/src/main/java/com/linecorp/centraldogma/client/spring/CentralDogmaSettings.java

```diff
@@ -222,13 +233,35 @@ public void setRetryIntervalOnReplicationLagMillis(@Nullable Long retryIntervalO
         this.retryIntervalOnReplicationLagMillis = retryIntervalOnReplicationLagMillis;
     }
 
+    /**
+     * Returns the number of milliseconds the Central Dogma client waits for initialization to complete.
+     * If {@code 0} is specified, Central Dogma client is immediately returned without waiting.
+     *
+     * @return {@code null} if not specified.
+     */
+    @Nullable
+    public Long initializationTimeoutMillis() {
```

Comment:
Not strong on this, but initialization sounds a little general (vs. `endpointReadyTimeoutMillis` or `endpointInitializationTimeoutMillis`)

Reply:
Because 1) I didn't want to expose the implementation detail and 2) we might add another logic later to warm up `CentralDogma` somehow.

Reply:
I see 👍  Makes sense